pkg/apis/openebs/v1alpha1/blockdevice_types.go

```diff
@@ -28,7 +28,10 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +k8s:openapi-gen=true
 
+// +kubebuilder:resource:scope=Namespaced,shortName=bd
+
 // BlockDevice is the Schema used to represent a BlockDevice CR
+// +kubebuilder:resource:scope=Namespaced,shortName=bd
```

Comment:
@harshthakur9030  Can you add the `additionalPrinterColumns`  also. ?

Reply:
@akhilerm I don't think it is required at CRD root level. I have provided it for other fields
https://book.kubebuilder.io/reference/markers/crd.html